### PR TITLE
fix typos, change link to website

### PR DESCRIPTION
**Pull request by nekomajin**
- fix typo: `e.g` -> `e.g.`
- fix typo: Capital letter at sentence start
- Please check my change on [line 63](https://github.com/gitterHQ/services/compare/master...nekomajin:patch-1?diff=unified&name=patch-1#diff-04c6e90faac2675aa89e2176d2eec7d8R63)

### CODE REVIEW

**Comment by MadLittleMods** [comment]
@nekomajin Please refrain from pinging certain people on the team when any one of us can review the PR.

**Comment by nekomajin** [comment]
Sorry. I know that as of other projects. I will improve myself :sweat_smile:

File: README.md [hunk 1]
```diff
@@ -36,36 +36,36 @@ If everything passes, then you are ready!
...
-      * `message`: (_string_) the message do be displayed (in markdown) e.g `"Some *Amazing* event has occured"`.
-      * `icon`: (_string_) the name of an icon in the `icons` dir to display e.g `"logo"`.
+      * `message`: (_string_) the message to be displayed (in markdown) e.g. `"Some *Amazing* event has occured"`.
+      * `icon`: (_string_) the name of an icon in the `icons` dir to display e.g. `"logo"`.
       * `errorLevel`: (_string_) either `"normal"` or `"error"`. Error messages get styled red, and so the `icon` that you pick for this message must be red too ([#e74c3c](http://www.colorhexa.com/e74c3c)).
 * `icons`: This directory contains all the png icons that can be used by this service. They must follow the following rules:
-  * each icon must exist as both a 16x16 png and a 32x32 png (`name.png` and `name@2x.png` respectively).
-  * there must be a logo icon (`logo.png` and `logo@2x.png`).
-  * icons must be either black and white, or [#e74c3c](http://www.colorhexa.com/e74c3c) and white (for error messages).
+  * Each icon must exist as both a 16x16 png and a 32x32 png (`name.png` and `name@2x.png` respectively).
```
Comment (MadLittleMods) [inline README.md]: Remove the capitalizations or make the other bullets uniform with this style

**Comment by MadLittleMods** [comment]
Merged via https://github.com/gitterHQ/services/commit/1737ef7949f274a5c88277e3f9b3c93bae19065f

Thanks @nekomajin 👍

**Pull request closed without merging**